README.md

```diff
@@ -42,20 +42,51 @@ You may also want to refer to the [Accedo One Rest API documentation](https://de
 
 - Include the library into your buildscript: 
 
+<details>
+<summary>Setup using Accedo Artifactory</summary>
+<p>
+
+```groovy
+// In root build.gradle
+maven {
+    // Accedo Control SDK in Artifactory
+    url "https://repo.cloud.accedo.tv/artifactory/control-sdk-android"
+    credentials { username = artifactory_user; password = artifactory_password }
+}
+
+// In project module
+dependencies {
+    implementation 'tv.accedo.one:control-sdk:<latest>'
+}
 ```
-repositories {
-	maven {
-		url  "https://dl.bintray.com/tibor-pasztor-accedo/accedo-products"
-	}
+
+</p>
+</details> 
+
+
+<details>
+<summary>Setup using Bintray (Which will be deprecated soon!)</summary>
```

Comment:
The Control SDK is public on github and can be used by 3rd parties. What are the plans, how they can access to it in the future?

Reply:
I will create a ticket for IT to check if we can allow anonymous access to repo (https://jfrog.com/knowledge-base/how-can-i-allow-anonymous-user-access-to-specific-repositories-only/)

Reply:
Cool, thank you!

Reply:
As they say, we won't serve artifacts anonymously :/ https://accedobroadband.jira.com/browse/IT-20515

Right now only way would be to open a repository in Sonatype but Idk if they are going to allow it 🤷 

If not, we serve aar